test/integration/forms/form.js

```diff
@@ -166,6 +166,9 @@ context('Patient Action Form', function() {
   });
 
   specify('storing stored submission', function() {
+    localStorage.clear();
+    // Just short of 5MB to cause quota error
+    localStorage.setItem('form-subm-too-big', _.times(5242800, _.constant('0')).join(''));
```

Comment:
If I set `_.times(100` here
<img width="382" alt="Screen Shot 2022-08-16 at 5 22 23 AM" src="https://user-images.githubusercontent.com/2028470/184712382-97786299-0a90-48ce-a64d-155a9e8390f6.png">

Reply:
whelp.. this test may not run on circleci.. it gets here and the test doesn't load..

Reply:
Ah dang ☹️